core/bug_api.php

```diff
@@ -851,6 +1069,45 @@ function bug_cache_row( $p_bug_id, $p_trigger_errors = true ) {
 }
 
 /**
+ * Determinates if we can add one more file attachment for this bug (as automated report). 
+ * @param integer $p_bug_id Identifies bug 
+ * @return boolean
+ * @access public
+ * @uses config_api.php
+ */
+function bug_allow_more_attaches( $p_bug_id ) {
+	$t_status = bug_get_field( $p_bug_id, 'status' );
+	$t_resolution = bug_get_field( $p_bug_id, 'resolution' );
+	
+	# Bug is closed for submission?
+	if( $t_status >= config_get( 'bug_collect_reports_status_threshold' ) ) {
+		return false;
+	}	
+	
+	# Bug is read-only?
+	if( $t_status >= config_get( 'bug_readonly_status_threshold' ) ) {
+		return false;
+	}	
+	
+	# Bug is closed?
+	if( $t_status >= config_get( 'bug_closed_status_threshold' ) ) {
```

Comment:
This is likely covered by the check for bug_collect_reports_status_threshold.  If the admin doesn't want to collect for closed bugs, then they can set bug_collect_reports_status_threshold to CLOSED which may be the default.